### PR TITLE
April 30th, Core WG Call Notes

### PR DESCRIPTION
**Pull request by brghena**
### Pull Request Overview

Adds Core Working Group call notes from April 30, 2021

### Testing Strategy

Spellcheck

### TODO or Help Wanted

Look over for quality

### Documentation Updated

- [X] Updated the relevant files in `/docs`, or no updates are required.

### Formatting

- [N/A] Ran `make prepush`.

### CODE REVIEW

**Comment by lschuermann** [comment]
Woah, that was fast. Thanks for taking notes in this involved discussion!

**Review by jrvanwhy — APPROVED** [review]

File: doc/wg/core/notes/core-notes-2021-04-30.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+ * Hudson: Even if we make it as fast as we can, there's going to be a fundamental limit.
+ * Phil: True, but a logarithmic tree search has the sorting question, but seems straightforward at least. I'd want to see the numbers though. I could totally be wrong about which way will win.
+ * Amit: I think the networking stack could be a decent litmus test for both. How reasonable would it be to benchmark on that?
+ * Leon: I imagine there will be different drivers looking at both ways. Console will use two allow operations each time.
+ * Amit: Actually, I think printf is non-blocking. Right?
+ * Leon: No, we can't modify the buffer while it's in flight.
+ * Gabe: printf definitely blocks. There is another put that is asynchronous.
+ * Amit: So if this is a question of performance, do we have a path to benchmarking?
+ * Hudson: Benchmarking the second is totally reasonable. Gives a cost for allow/unallow based on number of total AppSlices.
+ * Hudson: The second is tricky. The networking stack is probably a fine stress test.
```
Comment (lschuermann) [inline doc/wg/core/notes/core-notes-2021-04-30.md:84]:
```suggestion
 * Hudson: The first is tricky. The networking stack is probably a fine stress test.
```

I think this is referring to the first, aka. volatile / `UnsafeCell` approach.

**Review by lschuermann — COMMENTED** [review]
Looks good in general, but there might be one mixup of the two approaches we can take.

**Review by lschuermann — APPROVED** [review]

File: doc/wg/core/notes/core-notes-2021-04-30.md (hunk 1 above)
Comment (lschuermann) [reply doc/wg/core/notes/core-notes-2021-04-30.md]: It was a tough discussion to follow. :smile:

**Review by hudson-ayers — APPROVED** [review]

**Pull request merged**